addon/instance-initializers/load-feature-controls.js

```diff
@@ -1,21 +1,21 @@
-import { getStorage } from "ember-local-storage/helpers/storage"
 import config from "ember-get-config"
 
 const { featureControls } = config
 
 export function initialize(appInstance) {
   const features = appInstance.lookup("service:features")
   if (featureControls && featureControls.useLocalStorage) {
-    let featureControlsJSON = getStorage("local")["storage:feature-controls"]
-    if (featureControlsJSON) {
-      const flags = JSON.parse(featureControlsJSON)
-      if (flags) {
-        Object.keys(flags).forEach(flag => {
-          if (features.get("flags").includes(flag)) {
-            flags[flag] ? features.enable(flag) : features.disable(flag)
-          }
-        })
-      }
+
+    const controlStorageService = appInstance.lookup("service:feature-control-storage")
+    // result of controlStorageService.get('featuresLS') is an ObjectProxy we need to use "content"
+    let { content: featureControls } = controlStorageService.get('featuresLS')
+    if (featureControls) {
+      Object.keys(featureControls).forEach(flag => {
+        if (features.get("flags").includes(flag)) {
+          featureControls[flag] ? features.enable(flag) : features.disable(flag)
+        }
+      })
+      
     }
```

Comment:
Simple suggestion not necessary of course 👍🏻 .
- Without `if`
- Direct access to flags value

But I admit `Object.entries()` is less supported than `Object.keys`.

```suggestion
    let featureControls = controlStorageService.get('featuresLS').content ?? {}

    Object.entries(featureControls).forEach(([ flag, flagValue ]) => {
      if (features.flags.includes(flag)) {
        flagValue ? features.enable(flag) : features.disable(flag)
      }
    })
```